_layouts/global.html

```diff
@@ -147,7 +147,6 @@
           <li><a class="dropdown-item"
                  href="https://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
           <li><a class="dropdown-item" href="https://www.apache.org/foundation/thanks.html">Thanks</a></li>
-          <li><a class="dropdown-item" href="https://www.apache.org/security/">Security</a></li>
```

Comment:
Instead of removing this, please update it with the  project-specific one and match the regex ` ^https?://.*apache.org/.*[Ss]ecurity` to satisfy ASF requirements

Reply:
I think the point is that there is already a "Security" link in the Developer menu that points to the Spark Security page.

Reply:
I mean we can replace this with our security side to reach the goal of this PR and satisfy the ASF website check service

Reply:
I had assumed the Security link in the Developers menu would _already_ satisfy the ASF website check - but indeed I see the ASF website check (https://whimsy.apache.org/site/check/security) seems to want an absolute link to an apache.org site, while the Security link in the Developers menu is a relative link.

I guess we could fix this either on the spark-website side by replacing `{{site.baseurl}}` with `{{site.url}}{{site.baseurl}}` (see e.g. https://mademistakes.com/mastering-jekyll/site-url-baseurl/), or try to change the regex to also allow relative links.

Reply:
https://spark.apache.org/security.html can be used to satisfy both